### DISCUSSION TITLE
Visualizing temperatures

### DISCUSSION THREAD

**Discussion by warflash** [visgl/deck.gl#7115] [Q&A]
Hello 👋

I'm trying to create a weather map using deck.gl and I'm not sure how and if it's possible.
The ideal solution imo would be to supply a bunch of coordintes along a temperature to a heatmap layer, which instead of calculating the sum or mean interpolates between the temperatures (or generally speaking just any value) at each location. 
Another route that came to mind is prerendering an image that get's overlayed on top of the existing maptiles but that doesn't sound smart given the bandwidth/performance impact.
I appreciate any advice if maybe someone ran into something similar before!

**Comment by blanch0t** [comment]
As said [here](https://github.com/visgl/deck.gl/issues/4827#issuecomment-665307835), you should consider using [d3-delaunay](https://github.com/d3/d3-delaunay). Thankfully a custom layer using delaunay triangulation has already been made in [this example](https://github.com/visgl/deck.gl/tree/x/delaunay/examples/experimental/delaunay).
